.github/workflows/go-test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           - ubuntu-latest
           - macos-latest
         go-version:
-          - 1.19.x
```

Comment:
Go vet in 1.29 doesn't like `Unwrap() []error`. Till 1.20 only `Unwrap() error` was supported (without slice).

So I see two options, updating the minimum Go versions, or using build tags (`//go:build !go1.19`) so `Unwrap() []error` is only available starting in 1.20.

I think we are fine with supporting the last 3 minors, I am updating by now. @mrodm wdyt?

Reply:
> I think we are fine with supporting the last 3 minors, I am updating by now. @mrodm wdyt?

Yes, I agree with that. I think it is good enough to support here those 3 minors for now :+1: